src/openforms/js/components/admin/forms/Inputs.js

```diff
@@ -119,8 +119,8 @@ const Checkbox = ({name, label, helpText, noVCheckbox = false, ...extraProps}) =
 };
 
 Checkbox.propTypes = {
-  name: PropTypes.string.isRequired,
-  label: PropTypes.node.isRequired,
+  name: PropTypes.string,
+  label: PropTypes.node,
```

Comment:
with the remarked layout issues, I think the `name` and `label` really should stay required. Any form input element always should have a `name`, even if we don't really use it in the `onChange` handler, let's make sure we follow best HTML practices.